docs/resources/Guild.md

```diff
@@ -712,6 +721,9 @@ For guilds with [Membership Screening](#DOCS_RESOURCES_GUILD/membership-screenin
 > info
 > The Authorization header must be a Bot token (belonging to the same application used for authorization), and the bot must be a member of the guild with `CREATE_INSTANT_INVITE` permission.
 
+> info
+> This endpoint supports the `X-Audit-Log-Reason` header.
```

Comment:
Are u sure member add supports x-audit-log-reason?

Reply:
yeah this endpoint doesn't accept that header.